recipes/yamlloader/meta.yaml

```diff
@@ -0,0 +1,51 @@
+#{% set name = "yamlloader" %}
+{% set version = "0.5.2" %}
+{% set sha256 = "ea7b1e855d58cc80ba45c9d05566c7bbf3893d1335797a17d372145f81c0cb8a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - pyyaml
+  run:
+    - python
+    - pyyaml
+
+test:
+  imports:
+    - yamlloader
+
+about:
+  home: https://github.com/Phynix/yamlloader
+  license: MIT
+  license_family: MIT
+  summary: 'Loaders and dumpers for PyYAML'
+
+  description: |
+    This module provides loaders and dumpers for PyYAML. Currently, an OrderedDict
+    loader/dumper is implemented, allowing to keep items order when loading resp.
+    dumping a file from/to an OrderedDict.
+  dev_url: https://github.com/Phynix/yamlloader
+
+extra:
+  recipe-maintainers:
```

Comment:
Please confirm that all of the listed co-maintainers have agreed to be listed.

Reply:
Before you typically asked for the conda-forge people to be added (hmm, or am I thinking of bioconda now?). I haven't asked anyone other than mayou36, who is the author of the package. Should I remove the other ones?

Reply:
@mayou36 Please confirm :)

Reply:
Hey, thanks for adding it. I agree to be a maintainer